wp-content/themes/wp-starter/src/styles/base/global.css

```diff
@@ -15,3 +15,9 @@
 		}
 	}
 }
+
+@layer components {
+	.entry-content > * {
```

Comment:
No sure if there is a better more WP way to do this. But looked through the 2024 theme and did not find anything but hard code margins.

Reply:
I think this is OK for now, but I know that there's a margin setting on Group blocks, and by default it's not set to anything, but if you click it and drag it to 0, it removes margins. In the case of designs where there are 2 container blocks that should have no gap between them (like a hero, then a colored background group), this could be problematic. I'd say leave it for now, but we can explore this a bit more and maybe see if we can come up with a better solution.

Would this technically apply a top margin to the first block at the top of the page? So for example on IAFF, the video/hero element? We can always override it per-project, but also maybe just add `*:not(:first-child)`? I don't want to get too specific and make things more difficult, especially when we're using the `*` selector.